src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
+    },
+    goLeft(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+
+      this.coords.x -= value;
+
+      return this;
+    },
+    goBack(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+
+      this.coords.y -= value;
+
+      return this;
+    },
+    goRight(value = 1) {
```

Comment:
```suggestion
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    goForward(value = 1) {
      if (value < 0) {
        return this;
      }

      this.coords.y += value;

      return this;
    },
    
    goLeft(value = 1) {
      if (value < 0) {
        return this;
      }

      this.coords.x -= value;

      return this;
    },
    
    goBack(value = 1) {
      if (value < 0) {
        return this;
      }

      this.coords.y -= value;

      return this;
    },
    
    goRight(value = 1) {
```

Add blank line between methods